docs/User-Guide.md

```diff
@@ -52,44 +61,50 @@ _**Download and unpack/install the package to their own new folder!**_
-  * Unpack that ".tar" file once more into its own folder. 
-  * Run Forge app/exe
-* **Desktop Linux/Mac**:
-  * Unpack "forge...*tar.bz2*" with any unpacking app. (Check your package repository, or app store.)
-    * You'll probably end up with just a folder, and fully extracted.
-    * If you do end up with a ".tar" file, unpack that file also into its own folder.
-  * Run Forge script:
-    * Linux: Run the ".sh" file in a terminal (double clicking might work.)
-    * MacOS/OSX: Run the ".command" file by double clicking in Finder, or run from the terminal.
-       * If the command file doesn't appear to do anything, you'll need to [modify the permissions to be executable.](https://support.apple.com/guide/terminal/make-a-file-executable-apdd100908f-06b3-4e63-8a87-32e71241bab4/mac) (This is a temporary bug in the build process.)
-       * Additionally OSX needs to have a JRE AND a JDK installed because reasons. 
-* **Android**:
-  * Sideload/Install "forge...apk"
-  * Run Forge
-
-## Play Adventure Mode on Desktop
+### Manual Extraction (tar.bz2)
+
+#### Desktop Windows
+
+* Unpack "forge...*tar.bz2*" with any unpacking/unzipping app (e.g. 7-zip, winrar, etc)
+  * You'll end up with "forge...*tar*".
+* Unpack that ".tar" file once more into its own folder.
+* Run Forge app/exe
+
+#### Desktop Linux/Mac
+
+* Unpack "forge...*tar.bz2*" with any unpacking app. (Check your package repository, or app store.)
+  * You'll probably end up with just a folder, and fully extracted.
+  * If you do end up with a ".tar" file, unpack that file also into its own folder.
+* Run Forge script:
+  * Linux: Run the ".sh" file in a terminal (double clicking might work.)
+  * MacOS/OSX: Run the ".command" file by double clicking in Finder, or run from the terminal.
+    * If the command file doesn't appear to do anything, you'll need to [modify the permissions to be executable.](https://support.apple.com/guide/terminal/make-a-file-executable-apdd100908f-06b3-4e63-8a87-32e71241bab4/mac) (This is a temporary bug in the build process.)
+    * Additionally OSX needs to have a JRE AND a JDK installed because reasons.
+
+#### Android
+
+* Sideload/Install "forge...apk"
+* Run Forge
+
+### Play Adventure Mode on Desktop
 
 * Run the Adventure Mode EXE or Script in the Folder you extracted.
 * The game will start with an option for Adventure or Classic Mobile UI.
 * Android/Mobile builds are built as the Adventure Mode or Mobile UI and nothing special is needed.
-  - If adventure mode option does not show up;
-    - check you're up to date with your version.
-    - check in the settings that the "Selector Mode" is set to `Default`
+  * If adventure mode option does not show up;
+    * check you're up to date with your version.
+    * check in the settings that the "Selector Mode" is set to `Default`
+
+## Gameplay
 
-# Gameplay
+### Full Control
 
-## Full Control
+Right click/long tap on your player avatar:
 
-Right click/long tap on your player avatar:<br />
```

Comment:
FYI I'm actually using the html breaks in some places to only wrap to new line
your change will insert a full empty one between instead (which I think often looks ugly)

but I'm curious if you think it's better like this anyway? (for the rare chance someone only views it in text editor?!)

Reply:
Oh, then we can use something other than `<br>`. I personally prefer not having html tags inside .md files when possible.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#line-breaks